start_local_a+_env.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+git clone https://github.com/apluslms/course-templates.git my_new_course &&
+cd my_new_course &&
+git submodule init &&
+git submodule update &&
+./docker-compile.sh &&
+head -97 docker-up.sh > docker-up-custom.sh &&
+chmod +x docker-up-custom.sh &&
+ls -al &&
+docker ps &&
+netstat -tulpn &&
+./docker-up-custom.sh &&
+sleep 2m &&
+docker ps &&
+netstat -tulpn &&
+curl --user root:root http://localhost:8000/api/v2/courses/
```

Comment:
Nit: some comments here would be nice. Especially, why do we create a custom docker script with that `head` command (a bit hacky, not? ;) ). And `ls` is just for logging purposes, right? And what is the purpose of that 2m sleep. (I guess there was some discussion on that in Slack but can't remember exactly what.)

Reply:
comments added